Autopilot/Inc/airspeed.hpp

```diff
@@ -58,7 +58,7 @@ class airspeed {
  * Derived classes
  **********************************************************************************************************************/
 
-class MPXV7002DP : public airspeed {
+class MPXV7002DP : public Airspeed {
```

Comment:
I would not change the Airspeed class name now because it will mess up a lot of stuff in sensor fusion, etc. Keep it as was. We will change it later.

Reply:
<img width="1106" alt="Screen Shot 2021-06-11 at 8 10 15 PM" src="https://user-images.githubusercontent.com/52176616/121758858-29ac0300-caf1-11eb-98b7-03c4db392ce0.png">

Actually looking at this scratch what I said lol. Just need to change this class name

Reply:
sounds good, it has already started to break stuff lol